src/app.js

```diff
@@ -219,6 +228,36 @@ gpii.app.handleSessionStop = function (that, keyedOutUserToken) {
     }
 };
 
+/**
+ * Listen on uncaught exceptions and display it to the user is if it's interesting.
+ * @param that {Component} An instance of gpii.app.
+ */
+gpii.app.handleError = function (that) {
+    fluid.onUncaughtException.addListener(function (err) {
```

Comment:
what state? 
This is what I was aiming for to start with.. queryIoCSelector was what I wanted!